datahub-web-react/src/app/entityV2/shared/tabs/Dataset/Validations/contract/SchemaContractSummary.tsx

```diff
@@ -43,11 +43,11 @@ export const SchemaContractSummary = ({ contracts, showAction = false }: Props)
 
     const columns = [
         {
-            title: () => <ColumnHeader>NAME</ColumnHeader>,
+            title: () => <ColumnHeader>Name</ColumnHeader>,
```

Comment:
this aligns this table with our standard design pattern of giving table headers UpperCamelCase casing